src/core/locator/qgslocatorfilter.h

```diff
@@ -87,10 +89,20 @@ class CORE_EXPORT QgsLocatorResult
       * If left as empty string, this means that results are all shown without being grouped.
       * If a group is given, the results will be grouped by \a group under a header.
       * \note This should be translated.
-      * \since 3.2
+      * \since QGIS 3.2
       */
     QString group = QString();
 
+    /**
+      * Actions to be used in a context menu for the result.
+      * The key of the map is populated with IDs used to recognized
+      * entry when the result is triggered. The IDs should be 0 or greater
+      * otherwise, the result will be triggered normally.
+      * Entries in the context menu will be ordered by IDs.
+      * \since QGIS 3.6
+      */
+    QMap<int, QAction *> contextMenuActions = QMap<int, QAction *>();
```

Comment:
Can we replace this with the more generic "actions" name? I'm thinking specifically for QField and other clients, where the results may not be shown in a context menu but in some other way.

Reply:
(There's also no need to initialise this list here)